css/styles.css

```diff
@@ -0,0 +1,3 @@
+label {
+    display: block;
+}
```

Comment:
You did well by creating a separate file for styling. This is good because separating the file from the HTML code help to:

- improve readability of both files
- improve mantainability of code.
However, the styles.css file is too short. there is  a lot of styling that can be done to the form.